ProcessMaker/Models/User.php

```diff
@@ -148,6 +148,22 @@ public static function rules($existing = null)
         'permissions',
     ];
 
+    protected static function boot()
+    {
+        parent::boot();
+
+        // Setting defaults values for a new user
+        static::creating(function ($model) {
+            $appTimezone = getenv('APP_TIMEZONE') !== false ? getenv('APP_TIMEZONE') : 'America/Los_Angeles';
```

Comment:
No. These default values should be set in the DATABASE migration (as part of the database default values)